swap_meet/vendor.py

```diff
@@ -1,2 +1,91 @@
+from .item import Item
 class Vendor:
-    pass
+    def __init__(self, inventory=[]):
+        self.inventory = inventory
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item 
+    
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item 
+        else:
+            return False
+    
+    def get_by_category(self, category):
+        return [item for item in self.inventory if item.category == Item(category=category).category]
+    
+    def swap_items(self, Vendor, my_item, their_item):
+        if my_item in self.inventory and their_item in Vendor.inventory:
+            Vendor.remove(their_item)
+            self.add(their_item)
+            Vendor.add(my_item)
+            self.remove(my_item)
```

Comment:
we can take this a step further and merge this into 2 lines of code since the `remove` instance returns the item:

```suggestion
            self.add(vendor.remove(their_item))
            vendor.add(self.remove(my_item))
```